### DISCUSSION TITLE
Filename Conflict  - How to Overwrite Existing - doesn't seem to work? ..instead adds (1) (2) (3) and so on

### DISCUSSION THREAD

**Discussion by endeavorz** [gildas-lormeau/SingleFile#1339] [Q&A] [answered]
I see the option there under File Name> File Name Conflict Resolution> Overwrite The Existing File

..but it does not seem to work on my Win10 OS, it won't overwrite - it just keeps adding the next number (1) (2) (3) etc 

How to make it work? What am I missing here?

**Comment by gildas-lormeau** [comment, marked as answer]
Is the option `Misc. > save pages in background` enabled?

**Reply by endeavorz** [reply]
Yes Thank You, that fixed it.  
That setting is all the way down in a different place under Misc. I'm surprised they're not together or at least noted how one affects the other and is needed for the other to work? ...or maybe I just did not see it probably.
Anyway thank you for the answer, appreciate it.

**Reply by gildas-lormeau** [reply]
This setting should not be changed until you know what you're doing. That's why it's located in the Misc. section.

**Reply by endeavorz** [reply]
Ouch, well I never realized it was ever changed though and don't ever remember changing it <shrug>
